developer-docs/upgrading_kubernetes.md

```diff
@@ -120,7 +120,7 @@ We then create the "**stable**" RPMs. Follow the steps below.
 
 ### Release Notes
 
-Release notes should be drafted before the release is complete. This happens in the [Rancher Labs - Release Notes](https://github.com/rancherlabs/release-notes) repository. Create a new branch from your fork and update the relevant files in `rke2/`. The release note files have been standardized and should stay in the style they're currently in.
+Release notes should be drafted before the release is complete and ideally before the primary release tags are cut. This happens in the [Rancher Labs - Release Notes](https://github.com/rancherlabs/release-notes) repository. Create a new branch from your fork and update the relevant files in `rke2/`. The release note files have been standardized and should stay in the style they're currently in.
```

Comment:
We may want to reword as this can be a matter of preference as to when the release notes are completed. I prefer to do them after the full release tags are created while CI is running and before we uncheck the prerelease button.